src/AgGrid.fs

```diff
@@ -4,18 +4,200 @@ module Feliz.AgGrid
 open System
 open Fable.Core
 open Fable.Core.JsInterop
-
 open Feliz
 
 let agGrid : obj = import "AgGridReact" "ag-grid-react"
 
-importAll "ag-grid-community/styles/ag-grid.css"
-importAll "ag-grid-community/styles/ag-theme-alpine.css"
-importAll "ag-grid-community/styles/ag-theme-balham.css"
-importAll "ag-grid-community/styles/ag-theme-material.css"
+// User should load the CSS files in their own project

```

Comment:
Let's remove these comments and document that users need to do this in their apps. Would be a breaking change.